tests/tcp-test.py

```
@@ -1,18 +1,22 @@
 
 import meshtastic
 
+LOGGER = logging.getLogger(__name__)
 
-def onConnection(
-    interface, topic=pub.AUTO_TOPIC
-):  # called when we (re)connect to the radio
-    print(interface.myInfo)
+
+def on_connection(
+    interface: Any, _topic: Any = pub.AUTO_TOPIC
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `interface`

(ANN401)